app/models/financial_group.rb

```diff
@@ -0,0 +1,7 @@
+class FinancialGroup < ApplicationRecord
+  belongs_to :user
+  has_many :categorisations, dependent: :destroy
+  has_many :financial_entities, through: :categorisations
+  has_one_attached :icon
+  validates :name, presence: true
```

Comment:
- [ ] @0sugo Great work adding validation for the name kindly add the same for the icon, its part of te project requirement
![image](https://github.com/0sugo/cash-flow-nexus/assets/61408860/0b3d1048-5d65-48ac-8d44-1fe0bdc23c83)